lib/replace-selection-items.js

```diff
@@ -0,0 +1,14 @@
+import {splice} from 'immutable-array-methods';
+import getSelectionItems from './get-selection-items';
+
+export default (articleJson, fn) => {
+  const items = getSelectionItems(articleJson);
+
+  if (!items.length) {
+    return articleJson;
+  }
+
+  const selectionStartIndex = articleJson.indexOf(items[0]);
+  const deleteCount = items.length;
+  return splice(articleJson, selectionStartIndex, deleteCount, fn(items));
```

Comment:
Don't we want to do `splice(articleJson, selectionStartIndex, deleteCount, ...fn(items));` here? Right now we can only replace it with a single element - so we can implement `wrap` with this but not `unwrap` - for example.

Or am I reading the code wrong? Let's implement a test that does `unwrap` to be sure that use case works!